components/mixins/experience-event/events/changestatusinprogession.schema.json

```diff
@@ -0,0 +1,82 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/events/campaignstatusinprogression",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Campaign Status In Progression",
+  "type": "object",
+  "meta:conditionalField": "xdm:eventType",
+  "meta:conditionalValue": "leadOperation.campaignStatusInProgression",
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "Use for information regarding lead operation campaign status in progression events.",
+  "definitions": {
+    "campaignstatusinprogression": {
```

Comment:
Different from proposed name. Should always start with a verb to be consistent for all B2B experience events.